bootstrap/bootstrap.go

```diff
@@ -41,13 +41,13 @@ const (
 	// DefaultMaxMessageSize is the default maximum message size for gRPC responses
 	DefaultMaxMessageSize = 1024 * 1024 * 1024
 
-	// DefaultResourceExhaustedRetryDelay is the default delay between retries when the server returns
-	// a ResourceExhausted error.
-	DefaultResourceExhaustedRetryDelay = 100 * time.Millisecond
+	// DefaultRetryDelay is the default delay between retries when a gRPC request
+	// to one of the Access Nodes has errored out.
+	DefaultRetryDelay = 100 * time.Millisecond
 
-	// DefaultResourceExhaustedMaxRetryDelay is the default max request duration when retrying server
-	// ResourceExhausted errors.
-	DefaultResourceExhaustedMaxRetryDelay = 30 * time.Second
+	// DefaultMaxRetryDelay is the default max request duration when retrying failed
+	// gRPC requests to one of the Access Nodes.
+	DefaultMaxRetryDelay = 30 * time.Second
```

Comment:
Now that we are adding load-balancing functionality, should we maybe decrease the max retry duration?

Reply:
is this load balancing or failovers? I'm not aware of a demand for load balancing requests to different backends, but there is clear need for failing over when the primary is unavailable.